### ISSUE TITLE
Can't connect to livesplit anymore

### ISSUE THREAD

**Issue by Spongeday** [LiveSplit/LiveSplit#2235]
Since the last update when I try to connect my livesplit with a race on racetime.gg it doesn't work.
I can see the races, I choose my race, a pop up windows shows up but it stays in "loading mode" and then livesplit crashes.

**Comment by yatokami9** [comment]
Yeah, I keep getting the same. I can't connect Livesplit to racetime.gg anymore at all. It crashes everytime i try to connect it.

**Comment by wooferzfg** [comment]
Try re-downloading LiveSplit. If that doesn't work, please ask for help in the Racetime.gg Discord server.

**Comment by just-ero** [comment]
Does this issue persist on the most recent stable release of LiveSplit?
